test_samples.py

```diff
@@ -48,8 +48,13 @@ def main():
     if opts.list:
         print("\n".join(list_samples()))
     else:
+        print(args)
         url = args[0]
-        r = requests.get(url)
+        headers = {
+            'User-Agent': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)',
```

Comment:
It looks useful but do you mind submitting this change in a different PR. I think it needs some extra thoughts like setting it from an command line flag that defaults to "Mozilla/4.0..."

Reply:
@dangra Any thoughts on merging this PR, I am using this library in one of my projects, it will be very helpful for me if this feature is part of it.